src/Repository/NphOrderRepository.php

```diff
@@ -30,4 +30,17 @@ public function getOrdersByVisitType($participantId, $visitType): array
             ->getQuery()
             ->getResult();
     }
+
+    public function getOrdersBySampleGroup($participantId, $sampleGroup)
```

Comment:
Type-hint these parameters and provide a return type.  Also, this new public method should have a related test.